app/helpers/games_helper.rb

```diff
@@ -4,15 +4,14 @@ def render_piece(x, y)
 
     if piece == @current_piece
       content_tag(:div, image_tag(piece.icon_path, size: '48x48'), class: 'selected')
```

Comment:
@mscoutermarsh I keep getting nomethoderror on `nil`, when the view is trying to load all of the pieces. As I mentioned during the meeting the other day, this was already working to the point where I could see the "Save" button in all of the nil spaces, but now it's crashing at `nil`. Not sure what changed, but when I read the conditionals, I believe I'm accounting for `nil` with my last else.

Reply:
can you paste the error? Which line exactly